src/main/java/uk/gov/hmcts/probate/transformer/CallbackResponseTransformer.java

```diff
@@ -1450,8 +1450,25 @@ private void updateCaseBuilderForTransformCase(CaseData caseData, ResponseCaseDa
         builder
                 .ihtReferenceNumber(caseData.getIhtReferenceNumber())
                 .primaryApplicantAlias(caseData.getPrimaryApplicantAlias())
-                .solsExecutorAliasNames(caseData.getSolsExecutorAliasNames())
-                .solsDeceasedAliasNamesList(caseData.getSolsDeceasedAliasNamesList());
+                .solsExecutorAliasNames(caseData.getSolsExecutorAliasNames());
+
+        List<CollectionMember<AliasName>> deceasedAliasNames = EMPTY_LIST;
```

Comment:
think you can do:

List<CollectionMember<AliasName>> deceasedAliasNames = Optional.ofNullable(caseData.getDeceasedAliasNameList())
        .stream() etc then not need the null check